text/0110-track-reform.md

```diff
@@ -60,8 +62,53 @@ export default class Foo extends LightningElement {
 
 In the example above, calling `foo()` and `bar()` will always render the latest on the next tick, while calling `baz()` might or might not update on the next tick, depending on others mutations in the current job. With this reform, they will always get the latest on the next tick, no matter what.
 
+Also, because of the way is going to be implemented, those non decorated class properties will maintain identity. Example:
+
+```js
+export default class Foo extends LightningElement {
+    _x;
+
+    get x() {
+        this.x = value;
+    }
+
+    set x(value) {
+        this.x = value;
+    }
+}
+
+const bar = {};
+
+foo.x = bar;
+console.log(foo.x === bar); // true
+```
+
 ### Benefits
 
-* No need to learn what the `@track` decorator is and when to use it.
-* No need to ask whether a field should be tracked or not (we get this question a lot).
-* Deterministic update of the UI after mutating a field.
\ No newline at end of file
+* No need to learn the `@track` decorator until you really need it, which is to track mutations in the value of the field. Ex:
+
+```html
+<template>
+    <p>Your address is: {address.street}, {address.zipCode}</p>
+</template>
+```
+
+```js
+    // This statement mutates the field
+    // it will be reflected on the template by default
+    this.address = {
+        street: 'Mission St',
+        zipCode: '94102'
+    }
+
+    // this statament mutates the value of the field
+    // and it won't be reflected in the template.
+    // you need to use @track to properly reflect this in the template.
+    this.address.zipCode = '94104';
+```
+
+* Performance improvement: since there is no need to use proxies to observe primitive values (majority of use cases). Suppose we have a base component that is heavily used today (ex: icon) with a tracked property (ex: name, and it receives a string); by removing the track, which is not needed anymore for this field, we will remove the extra computations added by the proxy logic times how many instances of this component on the page.
+
+* Deterministic update of the UI after mutating a field.
```

Comment:
probably adding a little bit more here... saying that it is now deterministic because all fields are observed now.